crates/bdk/tests/psbt.rs

```diff
@@ -1,5 +1,6 @@
-use bdk::bitcoin::{Amount, FeeRate, Psbt, TxIn};
+use bdk::bitcoin::{FeeRate, Psbt, TxIn};
 use bdk::{psbt, KeychainKind, SignOptions};
+use bitcoin::Amount;
```

Comment:
Why this change?

Reply:
I have no idea, probably rust-analyzer stuff 🤔, fixed it.